orko-ui/src/containers/OpenOrdersContainer.js

```diff
@@ -59,7 +62,9 @@ class OpenOrdersContainer extends React.Component {
 
 function mapStateToProps(state, props) {
   return {
-    orders: getOrdersForSelectedCoin(state)
+    orders: this.props.showAll
```

Comment:
I doubt that will work - you need {{props.showAll}}

Reply:
oh, typo here.